### PR TITLE
MP-5886 FE Final adjustments

### PR DESCRIPTION
**Pull request by danilmoro3**
Ticket: https://spryker.atlassian.net/browse/MP-5886
CR: https://spryker.atlassian.net/browse/MP-5250

### CODE REVIEW

File: docs/marketplace/dev/front-end/ui-components-library/actions/actions-refresh-parent-table.md [hunk 1]
```diff
@@ -1,23 +1,26 @@
...
 ## Overview
 
 Actions Refresh Parent Table is an Angular Service that refreshes the data of the parent table of a Table in the current context.
-Check out this example below to see how to use Actions Refresh Parent Table service.
 
 {% info_block warningBox "Note" %}
 
 Make sure that the table opened from another table, for ex. in the Drawer.
 
 {% endinfo_block %}
```
Comment (FRITS-Kh) [inline docs/marketplace/dev/front-end/ui-components-library/actions/actions-refresh-parent-table.md:17]: Should it be as a code block?

File: docs/marketplace/dev/front-end/ui-components-library/data-transformers/object-map.md [hunk 2]
```diff
@@ -37,22 +42,15 @@ In the example below, the `datasource` will return an array with the transformed
...
+</spy-select>
 ```
 
-## Interfaces
-
+## Service registration
 
-
-`mapProps`—a Data Transformer that is set up with a configuration object.  
-`propName`—the name of the property from which the value needs to be transformed.
```
Comment (FRITS-Kh) [inline docs/marketplace/dev/front-end/ui-components-library/data-transformers/object-map.md:48]: Should we also keep the `propName` description?

File: docs/marketplace/dev/front-end/ui-components-library/data-transformers/pluck.md [hunk 3]
```diff
@@ -4,14 +4,18 @@ description: This document provides details about the Data Transformer Pluck ser
...
 
 ## Overview
 
 Data Transformer Pluck is an Angular Service that selects and returns a nested object by path via configuration.
+
 The `datasource` example below will return the value of the `three` key ('123') of the `data` input after receiving the response.
 
+Service configuration:
+
+`path` - the name of the property from which the value needs to be retrieved. The `path` may contain nested properties separated by dots, just like in a Javascript language.
```
Comment (FRITS-Kh) [inline docs/marketplace/dev/front-end/ui-components-library/data-transformers/pluck.md:17]:
```suggestion
`path` - the name of the property from which the value needs to be retrieved. The `path` may contain nested properties separated by dots, just like in JavaScript.
```

File: docs/marketplace/dev/front-end/table-design/table-column-types/index.md [hunk 4]
```diff
@@ -80,8 +81,17 @@ export class CustomTableColumnConfig {
   customOption? = 'customOption';
```
Comment (supproduction) [inline docs/marketplace/dev/front-end/table-design/table-column-types/index.md:81]: Please, add also an example with context interpolation

**Review by supproduction — CHANGES_REQUESTED** [review]

File: docs/marketplace/dev/front-end/ui-components-library/data-transformers/object-map.md (hunk 2 above)
Comment (danilmoro3) [reply docs/marketplace/dev/front-end/ui-components-library/data-transformers/object-map.md:48]: Reverted

**Review by supproduction — DISMISSED** [review]

**Review by FRITS-Kh — DISMISSED** [review]

**Review by gibmirmilch — APPROVED** [review]

**Pull request merged**